### PR TITLE
Feature nxp 23884 pipelines on it builds

### PR DESCRIPTION
**Pull request by nxmatic**

### CODE REVIEW

File: integration/Jenkinsfiles/ftest.groovy [hunk 1]
```diff
@@ -17,95 +17,94 @@
  *     jcarsique
  */
 
+@Library("nuxeo@feature-NXP-23884-pipelines-on-it-builds") _
```
Comment (jcarsique) [inline integration/Jenkinsfiles/ftest.groovy]: (remove before merge)

File: integration/Jenkinsfiles/ftest.groovy [hunk 2]
```diff
@@ -17,95 +17,94 @@
...
-                checkout(
-                    [$class: 'GitSCM',
-                        branches: [[name: '*/${BRANCH}']],
-                        browser: [$class: 'GithubWeb', repoUrl: 'https://github.com/nuxeo/nuxeo'],
-                        doGenerateSubmoduleConfigurations: false,
-                        extensions: [
-                            [$class: 'PathRestriction', excludedRegions: '', includedRegions: '''nuxeo-distribution/.*
+    
+    timestamps {
+        def sha = stage('clone') {
```
Comment (jcarsique) [inline integration/Jenkinsfiles/ftest.groovy]: nice :)

File: integration/Jenkinsfiles/ftest.groovy [hunk 3]
```diff
@@ -17,95 +17,94 @@
...
+            println error
+            throw error
+        } finally {
+            warningsPublisher()
+            claimPublisher()
+        }
+    }
+}
+
+def verifyClosure(String sha, String zipfile, String name, String dir, Closure post) {
```
Comment (jcarsique) [inline integration/Jenkinsfiles/ftest.groovy]: document shortly "emit code template"...

File: integration/Jenkinsfiles/ftest.groovy [hunk 4]
```diff
@@ -17,95 +17,94 @@
...
+                    failOnServerError('nuxeo-distribution/nuxeo-server-funkload-tests/target/tomcat/log/server.log')
+                },
+                'webdriver' : verifyClosure(sha, zipfile, 'webdriver', 'nuxeo-jsf-ui-webdriver-tests') {
+                    archive 'nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/**/failsafe-reports/*, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/*.png, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/*.json, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/**/*.log, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/**/log/*, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/**/nxserver/config/distribution.properties, nuxeo-distribution/nuxeo-server-cmis-tests/target/nxtools-reports/*, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/results/*/*'
+                    junit '**/target/surefire-reports/*.xml, **/target/failsafe-reports/*.xml, **/target/failsafe-reports/**/*.xml'
+                    failOnServerError('nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/tomcat/log/server.log')
+                }
+            )
+        } catch (Throwable error) {
+            println error
```
Comment (jcarsique) [inline integration/Jenkinsfiles/ftest.groovy]: good idea, I would wrap it around something like "`----- DEBUG catched Throwable -----`"

**Comment by pgmillon** [comment]
👍

**Pull request merged**